Sources/SmithyIdentity/AWSCredentialIdentityResolvers/StaticAWSCredentialIdentityResolver.swift

```diff
@@ -5,11 +5,17 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+@_spi(ClientConfigDefaultIdentityResolver) import protocol SmithyIdentityAPI.ClientConfigDefaultIdentityResolver
 import struct Smithy.Attributes
 
 /// A credential identity resolver that provides a fixed set of credentials
 public struct StaticAWSCredentialIdentityResolver: AWSCredentialIdentityResolver {
-    private let credentials: AWSCredentialIdentity
+    fileprivate let credentials: AWSCredentialIdentity
+
```

Comment:
The `@_spi` initializer below creates a AWS credential identity resolver with empty credentials.  This is used in internal clients, where the default AWS credential identity resolver is never used.